nft/nfts.yaml

```diff
@@ -649,6 +649,105 @@ paths:
                     type: boolean
                     description: 'Whether the given wallet owns any token in the given NFT collection.'
       operationId: isHolderOfCollection
+  '/{apiKey}/getNFTSales':
+    get:
+      summary: getNFTSales
+      description: Gets NFT sales that have happened through on-chain marketplaces
+      tags: ['NFT API Endpoints']
+      parameters:
+        - $ref: '#/components/schemas/tokenId'
+        - $ref: nfts_body.yaml#/startBlock
+          description: String - The block number to start fetching NFT sales data from. Allowed values are decimal integers and "latest". Defaults to "latest".
+        - $ref: nfts_body.yaml#/startLogIndex
+          description: Integer - The log index within the startBlock to start fetching NFT sales data from.
```

Comment:
let's add all descriptions into the components section below in case they need to be reused